src/PoissonSolvers/FFTOpenPoissonSolver.hpp

```diff
@@ -878,10 +953,17 @@ namespace ippl {
 
                     // apply proper normalization
                     for (unsigned int i = 0; i < Dim; ++i) {
-                        if (alg == Algorithm::VICO || alg == Algorithm::BIHARMONIC) {
-                            rho2_mr = rho2_mr * 2.0 * (1.0 / 4.0);
-                        } else {
-                            rho2_mr = rho2_mr * 2.0 * nr_m[i] * hr_m[i];
+                        switch (alg) {
+                            case Algorithm::HOCKNEY:
+                                rho2_mr = rho2_mr * 2.0 * nr_m[i] * hr_m[i];
+                                break;
+                            case Algorithm::VICO:
+                            case Algorithm::BIHARMONIC:
+                                rho2_mr = rho2_mr * 2.0 * (1.0 / 4.0);
+                                break;
+                            case Algorithm::DCT_VICO:
+                                rho2_mr = rho2_mr * (1.0 / 4.0);
+                                break;
```

Comment:
Add a default case. Actually, instead of this switch, you may access an array with the prefactors and the index is determined by the algorithm type.

Reply:
Of course the default is a must!

Reply:
What do you think the default should be? Shall we go with Hockney?

Reply:
I think as Matthias said the default should throw an error. I don't think the default should be Hockney as for readability it is better to have the Algorithm::HOCKNEY case.